cadc-storage-adapter/src/main/java/org/opencadc/inventory/storage/StorageClient.java

```diff
@@ -94,59 +90,38 @@
  * @author majorb
  *
  */
-public class StorageClient {
+public class StorageClient implements StorageAdapter {
```

Comment:
Does this class do anything beyond loading the StorageAdapter impl? It should probably just be replaced with something like a factory class that reads config and instantiates the impl.

Reply:
Ultimately, minoc's ArtifactAction will have to instantiate the impl and make it available to subclasses. That could be a single method there rather than another class... and probably /conf/minoc.properties which is read on startup (since you can't change the adapter impl at runtime) but different instances will need different impls.

Reply:
No, it's not very useful any longer.  I'll get rid of it.